Corona-Warn-App/src/main/res/layout/fragment_information.xml

```diff
@@ -9,49 +9,50 @@
         android:layout_height="match_parent"
         android:contentDescription="@string/information_title">
 
-        <include
+        <com.google.android.material.appbar.MaterialToolbar
             android:id="@+id/information_header"
-            layout="@layout/include_header"
-            android:layout_width="match_parent"
+            android:layout_width="0dp"
             android:layout_height="wrap_content"
-            app:icon="@{@drawable/ic_close}"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="parent"
-            app:layout_constraintBottom_toTopOf="@id/scrollview"
-            app:title="@{@string/information_title}" />
+            app:navigationIcon="@drawable/ic_close"
+            app:title="@string/information_title" />
 
         <ScrollView
             android:id="@+id/scrollview"
             android:layout_width="match_parent"
             android:layout_height="0dp"
-            android:fillViewport="true"
             android:paddingTop="@dimen/spacing_normal"
             android:paddingBottom="@dimen/spacing_normal"
-            android:clipToPadding="false"
             android:scrollbarStyle="outsideOverlay"
             app:layout_constraintBottom_toBottomOf="parent"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toBottomOf="@+id/information_header">
 
-            <LinearLayout
+            <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
FYI, this undoes https://github.com/corona-warn-app/cwa-app-android/pull/1740.

In https://github.com/corona-warn-app/cwa-app-android/pull/2947 I was asked to solve this using ConstraintLayout Flow, which would work here, too, because the parent of this layout is a `ConstraintLayout`, but I'm not really a fan of that

Reply:
Alright, I'll revert it back to `LinearLayout`. I first thought the issue might somehow be caused by it and I've left it like that.